Library/Homebrew/missing_formula.rb

```diff
@@ -36,8 +36,10 @@ def blacklisted_reason(name)
           You can read more about it at:
             #{Formatter.url("https://github.com/MacRuby/MacRuby")}
         EOS
-        when /(lib)?lzma/
-          "lzma is now part of the xz formula."
+        when /(lib)?lzma/ then <<~EOS
+          lzma is now part of the xz formula, and can be installed with:
+            brew install xz.
```

Comment:
Good idea but remove the trailing `.`

Reply:
Oh! not sure how that slipped in unnoticed. Will change 👍